cli/internal/publish/images/images.go

```diff
@@ -92,8 +123,9 @@ func ProcessDocument(ctx context.Context, c *cloudquery_api.ClientWithResponses,
 		}
 		item := item
```

Comment:
Since you're doing this, you might as well do `ims[listKey{name: item.Name, sum: item.Checksum}][0]` and then you don't have to write it twice.